virttest/remote_commander/remote_runner.py

```diff
@@ -544,6 +548,10 @@ def cmd_loop(self):
                     cmd = CmdSlave(self.read_msg()[1])
                     self.cmds[cmd.cmd_id] = cmd
                     try:
+                        # There is hidden bug. We can bump into condition when
+                        # we running some function-command. That function dump
+                        # some data to stdio. But, stdio is out of
+                        # consideration at this point. It can stuck.
```

Comment:
Yes, exactly. If remote_commander runs some function in sync mode it waits until the command finish. Command can use stderr, as it is not pulled it can hang. Again. This is not my concern. I just left notice. My patch doesn't change old behaviour. Please merge it.